src/commands/createFunction/scriptSteps/PythonScriptStep.ts

```diff
@@ -0,0 +1,48 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
```

Comment:
The Azure extensions use https://github.com/microsoft/vscode-azuretools/blob/main/utils/src/utils/AzExtFsExtra.ts instead of `fs-extra`.

Reply:
Updated.